cmake/modules/hunter_windows_env_command.cmake

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2015 Sumedh Ghaisas, Ruslan Baratov
+# All rights reserved.
+
+include(hunter_windows_to_snf_filename)
+
+#Set command to change environment variables to 8.3 filenames to windows
+function(hunter_windows_env_command variable_name)
+  
+  hunter_windows_to_snf_filename(${CMAKE_C_COMPILER} "CMAKE_C_COMPILER_SNF")
+  hunter_windows_to_snf_filename(${CMAKE_CXX_COMPILER} "CMAKE_CXX_COMPILER_SNF")
+  hunter_windows_to_snf_filename(${CMAKE_RC_COMPILER} "CMAKE_RC_COMPILER_SNF")
+  
+  set(${variable_name} "set" "cc=${CMAKE_C_COMPILER_SNF}" "&&" "set" "cxx=${CMAKE_CXX_COMPILER_SNF}" "&&" "set" "rc=${CMAKE_RC_COMPILER_SNF}" PARENT_SCOPE)
```

Comment:
> if you see the environment the build command is running in, it contains extra environment variables such as CC, CXX, RC etc.

Okay, that is the work for [clear-all.cmake](https://github.com/ruslo/hunter/blob/master/scripts/clear-all.cmake) module. There are already CC and CXX, I've added RC: https://github.com/ruslo/hunter/commit/c38cbb3178d01b14d777f67899abbf66e529c2aa

> Yes the code is needed

Unfortunatelly it will not work because of problems described in #246. The only correct way I know to implement this is to create CMake script wrapper and include `clear-all.cmake` there. Just like Qt do:
* https://github.com/ruslo/hunter/blob/master/cmake/schemes/url_sha1_qt.cmake.in (see `qt-build.cmake`
* https://github.com/ruslo/hunter/blob/master/scripts/qt-build.cmake.in